cdci_data_analysis/flask_app/dispatcher_query.py

```diff
@@ -565,9 +574,15 @@
 
     def generate_request_url_call_back(self, products_url, session_id, job_id) -> str:
         job_monitor_status_json_file = f'scratch_sid_{session_id}_jid_{job_id}/query_output.json'
+        # to be handled now, with the job_id generated taking into account only the user_id
+        job_monitor_status_json_file_aliased = f'scratch_sid_{session_id}_jid_{job_id}_aliased/query_output.json'
         request_url = ""
+        file = None
         if os.path.exists(job_monitor_status_json_file):
             file = open(job_monitor_status_json_file)
+        elif os.path.exists(job_monitor_status_json_file_aliased):
```

Comment:
hm, I think maybe aliased should take the priority if it exists? Since it is the one that corresponds to the currently processed request.
But then if this query_output is written in non-aliased directory at the moment, it should be written there.

Reply:
Good point.

However, I just realized that the path of the scratch directory in use is actually saved as a member of the InstrumentQueryBacvkEnd object..I guess I can simply use that one...will test neo